src/main.js

```diff
@@ -194,14 +197,22 @@ function configureCommandlineSwitchesSync(cliArgs) {
 				}
 			}
 
-			// Others
+			// Other 'enabled' flags
 			else if (argvValue === true || argvValue === 'true') {
 				if (argvKey === 'disable-hardware-acceleration') {
 					app.disableHardwareAcceleration(); // needs to be called explicitly
 				} else {
 					app.commandLine.appendSwitch(argvKey);
 				}
 			}
+
+			// ozone platform
+			else if (argvKey === 'ozone-platform') {
+				if (argvValue) {
+					app.commandLine.appendSwitch(argvKey, argvValue);
+					app.commandLine.appendSwitch('enable-features', 'UseOzonePlatform');
```

Comment:
The `UseOzonePlatform` feature no longer needs to be *explicitly* enabled since a couple of Electron releases ago, so this can safely be removed.

```suggestion
```